models/articles.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection.js");
+
+exports.getArticleById = (id) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = ${id}`).then(({rows})=> {

```

Comment:
This is not how your query should be structured. Remember to refer to the notes https://notes.northcoders.com/courses/js-back-end/node-postgres-with-express
![Screenshot 2023-11-20 at 12 13 53](https://github.com/Scodia619/nc-news/assets/150331972/87e2565c-82d6-4267-bfbd-0ad4a039f873)